docs/README.md

```diff
@@ -68,6 +68,7 @@ These guides provide additional support for meeting and working with the standar
 
 ### Processes
 
+- [Bypassing GitHub security controls](./processes/github_security_bypass.md)
```

Comment:
Not sure about the wording here as it implies we're trying to work around legitimate security controls as opposed to handling false postives.

I'm not sure what I'd say instead though.

"Resolving GitHub security alerts" or something better.

Reply:
Yeah I'm totally with you - when it came to it I couldn't immediately think of a good title so figured I'd get the reviewers to think of something for me 😂

Reply:
I've added some changes to use this better title